nncf/torch/dynamic_graph/patch_pytorch.py

```diff
@@ -343,16 +343,15 @@ def remove_private_functions(names: List[str]) -> List[str]:
         return filtered_names
 
     patched_namespace = get_namespace_to_extract_functions_from(namespace)
-    all_torch_function_names = []
     members = inspect.getmembers(patched_namespace)
-    for member in members:
-        if (
-            inspect.isfunction(member[1])
-            or inspect.isbuiltin(member[1])
-            or inspect.ismethod(member[1])
-            or inspect.ismethoddescriptor(member[1])
-        ):
-            all_torch_function_names.append(member[0])
+    all_torch_function_names = [
+        member[0]
+        for member in members
+        if inspect.isfunction(member[1])
```

Comment:
looked better with parenthesis for "if"

Reply:
https://github.com/openvinotoolkit/nncf/pull/3411/commits/72ed42ec0ec37ff945ecbcdcc7065cdc926013a7